pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -419,6 +421,59 @@ func getDiskTargetsForMigration(dom cli.VirDomain, vmi *v1.VirtualMachineInstanc
 	return copyDisks
 }
 
+func domXMLWithoutKubevirtMetadata(dom cli.VirDomain, vmi *v1.VirtualMachineInstance) (string, error) {
```

Comment:
Can we do something like this?
```
xml.Unmarshal(xmlstr, domStruct)
domStruct.metadata.kubevirt = nil
return xml.Marshal(domStruct)
```

Reply:
I actually wrote it like that initially, but it lost a whole bunch of information, probably because the struct is incomplete.
Editing the raw XML seemed to make the most sense, to ensure we keep everything exactly like libvirt wants it.

Reply:
Indeed. I agree. We will lose the parts of the XML that we didn't model in our struct but set by libvirt.

Reply:
Indeed. I agree. We will lose the parts of the XML that we didn't model in our struct but set by libvirt.

Reply:
Ah, I see that makes sense.